src/tcgwars/logic/impl/gen3/PopSeries3.groovy

```diff
@@ -401,29 +469,18 @@ public enum PopSeries3 implements LogicCardInfo {
           energyCost L, L
           attackRequirement {}
           onAttack {
-            damage 20
+            flip "Bustle", 2, {}, {},
```

Comment:
Couldn't this be simplified to just
```suggestion
            flip "Bustle", 2, {damage 20 applyAfterDamage CONFUSED}, {}
```
Or is there an instance before damage completes that getting confused a second time would be an issue? If so, setting a flag and using that to apply the confused status would also work. Using the map just doesn't seem entirely necessary to me.

Reply:
Yeah you are correct, this could be simplified as per your suggestion.  Thanks!